scenario/difinitions/definition.rb

```diff
@@ -0,0 +1,20 @@
+## update: test.rb
+module Animal
+  class Cat
+    def call
+      puts "meow"
+    end
+  end
+end
+
+Animal::Cat.new.call
+
+## definition: test.rb:9:1
+test.rb:(1,7)-(1,13)
+test.rb:(2,8)-(2,11)
```

Comment:
Currently, the definition returns two results on the master branch. I believe that only the first one is accurate. Is this correct?

Reply:
This is maybe because it detects not only `Animal` but also `Animal::Cat`. I should fix it, but it's a different problem.